dlt/destinations/impl/redshift/redshift.py

```diff
@@ -76,15 +78,22 @@ def run(self) -> None:
         credentials = ""
         if self._staging_iam_role:
             credentials = f"IAM_ROLE '{self._staging_iam_role}'"
+            if self._region_name:
+                credentials += f" REGION '{self._region_name}'"
         elif self._staging_credentials and isinstance(
             self._staging_credentials, AwsCredentialsWithoutDefaults
         ):
             aws_access_key = self._staging_credentials.aws_access_key_id
             aws_secret_key = self._staging_credentials.aws_secret_access_key
+            region_name = self._staging_credentials.region_name
             credentials = (
                 "CREDENTIALS"
-                f" 'aws_access_key_id={aws_access_key};aws_secret_access_key={aws_secret_key}'"
+                f" 'aws_access_key_id={aws_access_key};aws_secret_access_key={aws_secret_key}"
             )
+            if region_name:
```

Comment:
lets not append the region to the credentials, but make it a separate {region} template variable. 
i guess if its "" it won't affect the query

Reply:
So you think it's harder to read/maintain or do you see any other reason? :)

Reply:
jup, just because its cleaner and technically not part of the credential